### ISSUE TITLE
Monocular camera

### ISSUE THREAD

**Issue by xbcdbc** [RonaldSun/VI-Stereo-DSO#2]
Does the VI-DSO algorithm（Direct Sparse Visual-Inertial Odometry using Dynamic Marginalization） can work in Monocular camera？

**Comment by RonaldSun** [comment]
@xbcdbc 
Yes, that paper is a mono algorithm. To work in mono, original DSO initialization is needed with "stereo_weight" in bash file set to 0. I'll update the code to support mono later.

**Comment by xbcdbc** [comment]
> 
> 
> @xbcdbc
> Yes, that paper is a mono algorithm. To work in mono, original DSO initialization is needed with "stereo_weight" in bash file set to 0. I'll update the code to support mono later.

Thankyou so much. Waiting your suport-mono code.
